corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -312,14 +321,24 @@ cloudCare.CaseDetailsView = Backbone.View.extend(cloudCare.caseViewMixin).extend
     },
     
     render: function () {
-        var self = this;
+        var self = this,
+            $panelBody,
+            $panel;
         if (!self._everRendered) {
-            self.el = $('<section />').attr("id", "case-details").addClass("span5");
+            self.el = $('<section />').attr("id", "case-details").addClass("col-sm-5");
             self._everRendered = true;
         }
         $(self.el).html(""); // clear
+
+        $panel = $('<div class="panel panel-default"></div>')
+            .append('<div class="panel-heading">Case Details</div>');
```

Comment:
actually i forgot we do have a list of translated strings we pass into the backbone app that covers some of the buttons etc, but it's definitely not comprehensive